app/src/main/java/com/surrus/peopleinspace/glance/util/BaseGlanceAppWidgetReceiver.kt

```diff
@@ -0,0 +1,17 @@
+package com.surrus.peopleinspace.glance.util
+
+import androidx.glance.appwidget.GlanceAppWidget
+import androidx.glance.appwidget.GlanceAppWidgetReceiver
+import org.koin.core.component.KoinComponent
+
+abstract class BaseGlanceAppWidgetReceiver<T : BaseGlanceAppWidget<*>> : GlanceAppWidgetReceiver(),
+    KoinComponent {
+    override val glanceAppWidget: GlanceAppWidget
+        get() {
+            return createWidget().apply {
+                this.initiateLoad()
```

Comment:
I imagine this will evolve over time, particularly also for data that needs to be updated (e.g. if we wanted to show latests ISS position on the map, but this is a nice approach for now.